test/orm/declarative/test_basic.py

```diff
@@ -1229,6 +1229,28 @@ class Address(Base):
             in Address.__mapper__.attrs.user._reverse_property
         )
 
+    def test_join_with_callable(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

testing a review

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4831